src/module/system/text-editor.ts

```diff
@@ -944,10 +944,13 @@ async function augmentInlineDamageRoll(
                   ].flat(),
               );
 
+        // set inline damage traits to item:trait for the IWR system
+        const modifyTraits = traits ? traits.map((t) => `item:trait:${t}`) : [];
```

Comment:
I'm aware this needs altered to avoid messing with traits like materials and such.